comid/integrityregisters.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package comid
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/veraison/swid"
+)
+
+const TextType = "text"
+
+// IRegisterIndex is the interface to hold register index
+type IRegisterIndex interface{}
+
+// IntegrityRegisters holds the Integrity Registers
+type IntegrityRegisters struct {
+	Register map[IRegisterIndex]Digests
+}
+
+func NewIntegrityRegisters() *IntegrityRegisters {
+	reg := make(map[IRegisterIndex]Digests)
+	return &IntegrityRegisters{Register: reg}
```

Comment:
simpler and more idiomatically
```suggestion
	return &IntegrityRegisters{
		make(map[IRegisterIndex]Digests),
	}
```

Reply:
Made it `concise` but retained explicit setting to be clear